libs/core/src/lib/button/button.component.ts

```diff
@@ -69,6 +70,7 @@ export class ButtonComponent extends BaseButton implements OnChanges, CssClassBu
             this.fdType ? `fd-button--${this.fdType}` : '',
             this.compact ? 'fd-button--compact' : '',
             this.fdMenu ? `fd-button--menu` : '',
+            this._isDisabled() ? `is-disabled` : '',
```

Comment:
This can be a normal string, doesn't have to be a template literal (I know the `fd-button--menu` above is the same :) )